tutor/src/components/annotations/window-shade.jsx

```diff
@@ -29,11 +30,20 @@ export default class WindowShade extends React.Component {
     const { ux, children } = this.props;
 
     return (
-      <div className={`highlights-windowshade ${ux.isSummaryVisible ? 'down' : 'up'}`}>
-        <div className='centered-content'>
-          {children}
-        </div>
-      </div>
+      <ReactCSSTransitionGroup
+        component={(props) => React.Children.toArray(props.children)[0] || null}
+        transitionName="animate"
+        transitionEnterTimeout={300}
+        transitionLeaveTimeout={300}
+      >
+        {ux.isSummaryVisible &&
+          <div key="shade" className="highlights-windowshade">
+            <div className='centered-content'>
+              {children}
+            </div>
+          </div>
+        }
+      </ReactCSSTransitionGroup>
     );
```

Comment:
The `React.Children.toArray(props.children)[0]` seems kinda weird to me.  I think you're doing that so that the page doesn't have an unneeded div until it's needed?

It seems like we could make `WindowShade` be responsible for conditionally rendering the SummaryPage and then do something like:

```jsx
  <ReactCSSTransitionGroup  transitionName="animate" transitionEnterTimeout={300}  transitionLeaveTimeout={300}  className="highlights-windowshade">
         {ux.isSummaryVisible && <SummaryPage {...propsFromAnnotation} />}
  </ReactCSSTransitionGroup>
```

Thoughts on that?